BrightID-rn/src/store/index.js

```diff
@@ -1,10 +1,50 @@
 //store/index.js
 import thunkMiddleware from "redux-thunk";
-import { createStore, applyMiddleware } from "redux";
+import {createStore, applyMiddleware} from "redux";
 import reducer from "../reducer";
+import {setPPKeys} from "../actions/index";
+import nacl from "tweetnacl"
+import axios from "axios"
 
 const store = createStore(reducer, applyMiddleware(thunkMiddleware));
 
+export const setupPPKeys = ppKeys => async dispatch => {
+
+    try {
+
+        if (ppKeys.hasOwnProperty("key1") && ppKeys.hasOwnProperty("key2")) {
+            dispatch(setPPKeys(ppKeys));
+        }
+    } catch (err) {
+        console.error(err);
+    }
+};
+
+export const generatePPKeys = () => async dispatch => {
+
+    try {
+
+        key1 = nacl.box.keyPair();
```

Comment:
Use  [```nacl.sign.keyPair()```](https://github.com/dchest/tweetnacl-js#naclsignkeypair) instead of ```box.keyPair()```

Reply:
Will do.